### PR TITLE
[PaddleNLP] Address late comments in summarization

### PR DESCRIPTION
**Pull request by sijunhe**
Address comments from @osanseviero which came late in #160

### CODE REVIEW

**Review by Narsil — APPROVED** [review]

**Comment by Narsil** [comment]
Waiting for @osanseviero this time :D

File: docker_images/common/app/pipelines/summarization.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __call__(self, inputs: Union[str, List[str]]) -> List[Dict[str, str]]:
         Args:
             inputs (:obj:`str`): a string to be summarized
         Return:
-            A :obj:`dict`:. The object return should be like {"summary_text": "The string after summarization"}
+            A :obj:`list` of :obj:`dict` in the form of {"summary_text": "The string after summarization"}
```
Comment (osanseviero) [inline docker_images/common/app/pipelines/summarization.py:21]: Please change the input type in line 16 to be just a `str`, not `List[str]`

File: docker_images/paddlenlp/tests/test_api_summarization.py [hunk 2]
```diff
@@ -50,19 +50,3 @@ def test_single_input(self):
         self.assertEqual(len(content), 1)
         for result in content:
             self.assertIn("summary_text", result)
-
-    def test_batch_input(self):
```
Comment (osanseviero) [inline docker_images/paddlenlp/tests/test_api_summarization.py:54]: Please also do this change in the common/ examples directory

**Review by osanseviero — COMMENTED** [review]
Thanks a lot for the follow-up PR!

**Comment by sijunhe** [comment]
done

**Review by osanseviero — APPROVED** [review]
Awesome, thanks a lot! :fire:

**Pull request merged**